### ISSUE TITLE
index.html references non-existant Launcher file

### ISSUE THREAD

**Issue by twistedpair** [sjrd/scala-js-example-app#18]
I'm new to Scala.js, though I think I found a bug. Running through the examples in the README.md, the tutorial works, but the referenced `example-launcher.js` file does not exist in target.

```html
<script type="text/javascript" src="./target/scala-2.12/example-launcher.js"></script>
```

Since the tutorial appears to work without it, and since the current [Scala.js intro tutorial](https://www.scala-js.org/tutorial/basic/) also lacks said file, I'm wondering if it's vestigial. Also, cursory reading/hacking indicates `fastOptJS` and `persistLauncher := true` configurations that would create the launcher file are exclusive.

**Comment by sjrd** [comment]
Yes, it is vestigial. References to the launcher should be removed.

**Comment by sjrd** [comment]
Fixed in #19.
